src/Features/LanguageServer/Protocol/Handler/Diagnostics/DocumentPullDiagnosticHandler.cs

```diff
@@ -35,14 +34,26 @@ internal class DocumentPullDiagnosticHandler : AbstractPullDiagnosticHandler<Doc
             => diagnosticsParams.TextDocument;
 
         protected override DiagnosticReport CreateReport(TextDocumentIdentifier? identifier, VSDiagnostic[]? diagnostics, string? resultId)
-            => new DiagnosticReport { Diagnostics = diagnostics, ResultId = resultId };
+            => new DiagnosticReport
```

Comment:
Should we set an Identifier here just in case? I worry that the workspace report technically has `Supersedes = 0`, and this has an identifier of 0, and whilst I'm sure the platform handles that, it still makes me nervous.

Reply:
sure.  i dont' mind being explicit.